lib/commands/web.js

```diff
@@ -160,15 +160,18 @@ async function tapWebElementNatively (driver, atomsElement) {
     }
 
     if (text) {
-      const el = await driver.findNativeElementOrElements('accessibility id', text, false);
-      // use tap because on iOS 11.2 and below `nativeClick` crashes WDA
-      const rect = await driver.proxyCommand(`/element/${el.ELEMENT}/rect`, 'GET');
-      const coords = {
-        x: Math.round(rect.x + rect.width / 2),
-        y: Math.round(rect.y + rect.height / 2),
-      };
-      await driver.clickCoords(coords);
-      return true;
+      const els = await driver.findNativeElementOrElements('accessibility id', text, true);
+      if (els.length === 1) {
```

Comment:
So we just do nothing instead if multiple elements are found? I wouldn't say it's very obvious behaviour

Reply:
We also can't assume that if there are multiple elements with the same text the first one is the one we want, it gives a false positive whereas falling back to translating the coords will always tap the element we want.